### PR TITLE
chore(stages): reduce the progress logging 

### PR DESCRIPTION
**Pull request by jsvisa**
There're many similiar `Writing xxx` logs in my reth node in a short time, after that node was restarted:

```
2024-10-08T15:53:16.587632Z  INFO Loading indices into database
2024-10-08T15:53:16.640678Z  INFO Writing indices progress=0.99%
2024-10-08T15:53:16.673921Z  INFO Writing indices progress=1.98%
2024-10-08T15:53:16.694879Z  INFO Writing indices progress=2.97%
2024-10-08T15:53:16.712707Z  INFO Writing indices progress=3.97%
2024-10-08T15:53:16.723617Z  INFO Writing indices progress=4.96%
2024-10-08T15:53:16.729454Z  INFO Writing indices progress=5.95%
2024-10-08T15:53:16.729891Z  INFO Writing indices progress=6.94%
2024-10-08T15:53:16.730354Z  INFO Writing indices progress=7.93%
2024-10-08T15:53:16.731004Z  INFO Writing indices progress=8.92%
2024-10-08T15:53:16.738051Z  INFO Writing indices progress=9.92%
2024-10-08T15:53:16.763732Z  INFO Writing indices progress=10.91%
2024-10-08T15:53:16.787495Z  INFO Writing indices progress=11.90%
2024-10-08T15:53:16.809150Z  INFO Writing indices progress=12.89%
2024-10-08T15:53:16.832691Z  INFO Writing indices progress=13.88%
2024-10-08T15:53:16.856580Z  INFO Writing indices progress=14.87%
2024-10-08T15:53:16.884020Z  INFO Writing indices progress=15.86%
2024-10-08T15:53:16.922661Z  INFO Writing indices progress=16.86%
2024-10-08T15:53:16.949469Z  INFO Writing indices progress=17.85%
2024-10-08T15:53:16.968421Z  INFO Writing indices progress=18.84%
2024-10-08T15:53:16.971475Z  INFO Writing indices progress=19.83%
2024-10-08T15:53:16.991315Z  INFO Writing indices progress=20.82%
...
```

so I propose to logging the progress by a period interval, instead of the count(eg: at most once per 5s).

### CODE REVIEW

**Review by onbjerg — APPROVED** [review]
makes sense to me

**Review by joshieDo — APPROVED** [review]
i'd be fine with more than 5s as well tbh

File: crates/stages/stages/src/stages/tx_lookup.rs [hunk 1]
```diff
@@ -147,16 +148,19 @@ where
                     .cursor_write::<tables::RawTable<tables::TransactionHashNumbers>>()?;
 
                 let total_hashes = hash_collector.len();
-                let interval = (total_hashes / 10).max(1);
+                let log_interval = Duration::from_secs(5);
```
Comment (mattsse) [inline crates/stages/stages/src/stages/tx_lookup.rs:151]: I'd like to make this a constant as well

**Review by mattsse — DISMISSED** [review]
pedantic nit

File: crates/stages/stages/src/stages/tx_lookup.rs (hunk 1 above)
Comment (jsvisa) [reply crates/stages/stages/src/stages/tx_lookup.rs:151]: good catch, I'll reuse the LOG_INTERVAL defined in utils.rs

**Pull request merged**